kaltura_admin.py

```diff
@@ -34,74 +38,100 @@ def get_kaltura_session(user_id, partnerID, adminkey):
     return client
 
 
-"""
-Get the configuration file
-"""
-parser = argparse.ArgumentParser(description='Generate Kaltura KMC admin list')
-parser.add_argument('env', type=str, help='env json file with kmc settings')
-args = parser.parse_args()
-if not os.path.isfile(args.env):
-    log.warn("env.json file does not exist. Please provide file path.")
-    sys.exit()
+def get_kmc_admin_list(kaltura_user_id, kmcs):
+    """
+    loop through all Kaltura Management Consoles
+    looking for people with admin roles
+    """
+    column_names = ['kmc_name', 'user_id',
+                    'first_name', 'last_name', 'role_name']
+    user_df = pd.DataFrame(columns=column_names)
+    for kmc_name in kmcs:
```

Comment:
nitpick: This would be simpler as 

`for kmc_name, kmc in kmcs.items():`

And just removing line 50. I might also rename. Nicer as a method though.

Reply:
Done.